packages/react/src/helpers/utils.ts

```diff
@@ -60,3 +61,57 @@ export const getFormDataFromEvent = (
   const formData = new FormData(event.target as HTMLFormElement);
   return Object.fromEntries(formData);
 };
+
+interface fieldProps {
+  labelHidden: boolean;
+  isRequired?: boolean;
+  label: string;
+  placeholder: string;
+  required?: boolean; // TextField input sometimes uses required instead of isRequired
```

Comment:
We should only use `isRequired`, it's a typo to use `required`. Can we remove this and only use `isRequired`?

Reply:
OK! Will change to `isRequired` for React here.

Reply:
Updated in 9604434331484fb8ac9992e549caaf99d5dba581